app/jobs/create_organization_report_job.rb

```diff
@@ -0,0 +1,33 @@
+class CreateOrganizationReportJob < ApplicationJob
+  queue_as :default
+
+  def perform(id:)
+    organization = Organization.find(id)
+    return unless organization
+    campaigns = organization.campaigns&.active
+    return unless campaigns
```

Comment:
This will be an instance of an ActiveRecord Relation. Probably better to check `campaigns.exists?` for early exit.